src/fidesops/ops/api/v1/endpoints/masking_endpoints.py

```diff
@@ -29,16 +27,16 @@ def mask_value(request: MaskingAPIRequest) -> MaskingAPIResponse:
     """Masks the value(s) provided using the provided masking strategy"""
     try:
         values = request.values
-        masking_strategy = request.masking_strategy
-        strategy = MaskingStrategyFactory.get_strategy(
-            masking_strategy.strategy, masking_strategy.configuration
+        masking_strategy_spec = request.masking_strategy
+        masking_strategy: MaskingStrategy = strategy(  # type: ignore
```

Comment:
i don't like having to suppress the type checker here, but i couldn't find a better way to get around this. by using a single, unified factory, we lose the ability to strongly type the return type of the `strategy` getter -- since the factory only knows about the abstract `Strategy` class, it must return a `Strategy` here and not a more specific subclass. i couldn't find a clean way for the calling ("client") code to specify what subtype of `Strategy` it was expecting back.